source/extensions/all_extensions.bzl

```diff
@@ -28,3 +28,13 @@ def envoy_windows_extensions():
         windows_extensions.append(path)
 
     return windows_extensions
+
+def envoy_ppc_extensions():
+   all_extensions = envoy_all_extensions()
+   luaijt_path = ''
+   for extension_path in all_extensions:
+      if extension_path.find("lua") > 0:
+         luajit_path = extension_path
+   if luajit_path != '':
+      all_extensions.remove(luajit_path)
```

Comment:
Ideally you use the suggested constant in `repositories.bzl` here (single source of truth). Also, I think this is something we'll want for other architectures/platforms (ARM), the ability to blacklist extensions. So, I would define a macro like: `def is_blacklisted_extension(extension_name, extension_blacklist):` to factor this logic out.